CHANGELOG.md

```diff
@@ -4,6 +4,41 @@ For advice on how to use these release notes see [our guidance on staying up to
 
 ## Unreleased
 
+#### Update the HTML for the character count
+
+We've updated the HTML for the character count.
+
+The component wrapper `data-module="govuk-character-count"` and its form group `class="govuk-form-group"` are now combined as the same `<div>`. The hint text used as the count message is now placed directly after the `<textarea>`.
+
+If you're not using Nunjucks macros, then you should:
+
+- move the `govuk-form-group` class from the form group to the component wrapper `<div>`
+- move the `govuk-form-group--error` class (optional) from the form group to the component wrapper `<div>`
+- move any other custom classes and attributes from the form group to the component wrapper `<div>`
+- remove the opening `<div>` and closing `</div>` tags previously used by the form group
+
+For example, see how the HTML below:
```

Comment:
I'd change the explanatory text so it isn't split between the code examples. Maybe say something like 'The following example shows some HTML and the difference once it is updated.' Then you can always add something like 'HTML before:' and 'HTML after:' to clearly divide them.